EVEStandard/API/Dogma.cs

```diff
@@ -32,6 +31,15 @@ internal Dogma(string dataSource) : base(dataSource)
             return returnModelDTO<DogmaAttribute>(responseModel);
         }
 
+        public async Task<ESIModelDTO<DogmaDynamicItem>> GetDynamicItemInfoV1Async(long typeId, long itemId, string ifNoneMatch = null)
+        {
+            var responseModel = await GetAsync("/v1/dogma/dynamic/items/" + typeId + "/" + itemId, ifNoneMatch);
```

Comment:
I really liked string interpolation, something I didn't know about before NoDime had it in a PR. I'd like anything new we add to the code to use it instead of string concat so this line would change to: 

```
var responseModel = await GetAsync($"/v1/dogma/dynamic/items/{typeId}/{itemId}", ifNoneMatch);
```